### ISSUE TITLE
Failed to build ImageMagick on Julia 1.0.2.1 - Ubuntu 18.04

### ISSUE THREAD

**Issue by shabbir-hasan** [JuliaIO/ImageMagick.jl#147]
Building the package fails on my Ubuntu 18.04 system

```
(JuliaPro_v1.0.2.1) pkg> add https://github.com/JuliaIO/ImageMagick.jl
  Updating registry at `~/.juliapro/registries/JuliaPro`
  Updating git-repo `https://pkg.juliacomputing.com/registry/JuliaPro`
  Updating git-repo `https://github.com/JuliaIO/ImageMagick.jl`
 Resolving package versions...
  Updating `~/JuliaPro-1.0.2.1/JuliaPro/pkgs-1.0.2.1/environments/JuliaPro_v1.0.2.1/Project.toml`
 [no changes]
  Updating `~/JuliaPro-1.0.2.1/JuliaPro/pkgs-1.0.2.1/environments/JuliaPro_v1.0.2.1/Manifest.toml`
 [no changes]
  Building ImageMagick → `~/.juliapro/packages/ImageMagick/at2AC/deps/build.log`
┌ Error: Error building `ImageMagick`: 
│ ┌ Warning: platform_key() is deprecated, use platform_key_abi() from now on
│ │   caller = ip:0x0
│ └ @ Core :-1
│ [ Info: JuliaTeam: redirecting https://github.com/bicycle1885/ZlibBuilder/releases/download/v1.0.2/Zlib.v1.2.11.x86_64-linux-gnu.tar.gz to https://pkg.juliacomputing.com/binary/ZlibBuilder/v1.0.2/Zlib.v1.2.11.x86_64-linux-gnu.tar.gz
│ ERROR: LoadError: LoadError: Could not unpack /home/enchanter/.juliapro/packages/ImageMagick/at2AC/deps/usr/downloads/libpng.v1.6.31.x86_64-linux-gnu.tar.gz into /home/enchanter/.juliapro/packages/ImageMagick/at2AC/deps/usr
│ Stacktrace:
│  [1] error(::String) at ./error.jl:33
│  [2] #unpack#98(::Bool, ::Function, ::String, ::String) at /home/julia/worker/juliapro-release-centos7-0_6/build/tmp_julia/share/julia/stdlib/v1.0/BinaryProvider/src/PlatformEngines.jl:667
│  [3] #unpack at ./none:0 [inlined]
│  [4] macro expansion at ./logging.jl:320 [inlined]
│  [5] #install#133(::Prefix, ::String, ::Bool, ::Bool, ::Bool, ::Function, ::String, ::String) at /home/julia/worker/juliapro-release-centos7-0_6/build/tmp_julia/share/julia/stdlib/v1.0/BinaryProvider/src/Prefix.jl:319
│  [6] (::getfield(BinaryProvider, Symbol("#kw##install")))(::NamedTuple{(:prefix, :force, :verbose),Tuple{Prefix,Bool,Bool}}, ::typeof(install), ::String, ::String) at ./none:0
│  [7] top-level scope at /home/enchanter/.juliapro/packages/ImageMagick/at2AC/deps/build_libpng.v1.0.0.jl:36
│  [8] include at ./boot.jl:317 [inlined]
│  [9] top-level scope at /home/enchanter/.juliapro/packages/ImageMagick/at2AC/deps/build.jl:15 [inlined]
│  [10] top-level scope at ./none:0
│  [11] include at ./boot.jl:317 [inlined]
│  [12] include_relative(::Module, ::String) at ./loading.jl:1044
│  [13] include(::Module, ::String) at ./sysimg.jl:29
│  [14] include(::String) at ./client.jl:392
│  [15] top-level scope at none:0
│ in expression starting at /home/enchanter/.juliapro/packages/ImageMagick/at2AC/deps/build_libpng.v1.0.0.jl:32
│ in expression starting at /home/enchanter/.juliapro/packages/ImageMagick/at2AC/deps/build.jl:11
│ tar: ./share: Cannot unlink: Directory not empty
│ tar: ./share/man: Cannot unlink: Directory not empty
│ tar: ./share/man/man3: Cannot unlink: Directory not empty
│ tar: ./include: Cannot unlink: Directory not empty
│ tar: ./manifests: Cannot unlink: Directory not empty
│ tar: ./logs: Cannot unlink: Directory not empty
│ tar: ./lib: Cannot unlink: Directory not empty
│ tar: ./lib/pkgconfig: Cannot unlink: Directory not empty
│ tar: Exiting with failure status due to previous errors
```
I did some investigation on my own but didn't manage to fix the problem (started using Julia a few days ago).

**Comment by balayan01** [comment]
got the same Problem
>julia
>using Pkg
>Pkg.build("ImageMagick")

run the last command 2-3 times if doesn't work in one go..

**Comment by timholy** [comment]
Should be fixed on Julia 1.3
